dockers-install.sh

```diff
@@ -37,3 +37,11 @@ else
   sudo curl -fsSL https://download.docker.com/linux/debian/gpg -o /etc/apt/keyrings/docker.asc; sudo chmod a+r /etc/apt/keyrings/docker.asc
 fi
 
+if (test -f /etc/bin/keyrings/docker.asc)
```

Comment:
you should check out how to test for the correct mode here, something like:

`stat -c "%a" /etc/bin/keyrings/docker.asc`

Reply:
And then make sure you get the number you want, based on the permissions for `a+r`.

Reply:
- I will work on that tonight, I am Imaging a fresh SD card now. Thank you sir!

Reply:
- Also found this earlier today, and it is custom to PI OS lite 64, which what I am using.
```curl -fsSL https://download.docker.com/linux/debian/gpg | sudo -H gpg --dearmor -o /etc/apt/keyrings/docker.gpg```

Reply:
- The issue is the .asc or .gpg which is what the `chmod` command is doing with the command above It is testing out now
- https://forums.docker.com/t/save-docker-repository-signing-key-correctly-in-asc-format/128373

Reply:
- I Think I got it, but then the terminal freezes
- Current readout
```chris@docker:~ $ bash test-docker.sh 
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 http://deb.debian.org/debian-security bookworm-security InRelease
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease    
Hit:4 http://archive.raspberrypi.com/debian bookworm InRelease   
Get:5 http://deb.debian.org/debian bookworm/main Sources [9,489 kB]
Get:6 http://deb.debian.org/debian bookworm/non-free Sources [77.9 kB]                                    
Get:7 http://deb.debian.org/debian bookworm/non-free-firmware Sources [6,168 B]                           
Get:8 http://deb.debian.org/debian bookworm/contrib Sources [51.3 kB]                                     
Get:9 http://deb.debian.org/debian-security bookworm-security/contrib Sources [856 B]                     
Get:10 http://deb.debian.org/debian-security bookworm-security/main Sources [90.8 kB]                     
Get:11 http://deb.debian.org/debian-security bookworm-security/non-free-firmware Sources [796 B]          
Get:12 http://deb.debian.org/debian bookworm-updates/non-free-firmware Sources [2,076 B]                  
Get:13 http://deb.debian.org/debian bookworm-updates/contrib Sources [776 B]                              
Get:14 http://deb.debian.org/debian bookworm-updates/non-free Sources [1,544 B]                           
Get:15 http://deb.debian.org/debian bookworm-updates/main Sources [17.4 kB]                               
Fetched 9,738 kB in 11s (866 kB/s)                                                                        
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
3 packages can be upgraded. Run 'apt list --upgradable' to see them.
Installing ca-certificates
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
ca-certificates is already the newest version (20230311).
0 upgraded, 0 newly installed, 0 to remove and 3 not upgraded.
/usr/bin/curl
curl already installed
Creating keyrings
/usr/bin/gpg
gpg keyring exists, and configured
configuring gpg keyrings
File '/etc/apt/keyrings/docker.gpg' exists. Overwrite? (y/N) y
```

Reply:
After i press y, no prompt, and I can type in the blank area.

Reply:
I am going to push the code up, so you can see what I did.